bin/build.sh

```diff
@@ -130,13 +130,13 @@ if [[ "${build}" = true || "${build_js}" = true ]]; then
       sudo gem install compass
     fi
   fi
-  (cd ./modules/ui && npm install && bower install && grunt build)
+  (cd ./modules/ui && npm install && bower install && grunt --forcce build)
```

Comment:
thx ... was having trouble building dist w/o --force ... at one time.

but, given there is a typo ... i will give it a go w/o --force

re "/target/" can you ref a line?

in general, it is desirable/optimal to put gen'd files and such in a "target" dir so that it is not git added, cleaned up with 'mvn clean', etc. does this help?